### PR TITLE
adding RP emulator

### PR DESCRIPTION
**Pull request by GG23800**

### CODE REVIEW

**Review by clecoued — CHANGES_REQUESTED** [review]
folder name should be renamed from RedPitaya_emulator to Probe_emulator

File: RedPitaya_emulator/srcbin/stepper.h [hunk 1]
```diff
@@ -0,0 +1,228 @@
+#ifndef STEPPER_H
```
Comment (clecoued) [inline RedPitaya_emulator/srcbin/stepper.h]: the stepper file is not needed in probe_emulator

File: RedPitaya_emulator/srcbin/emul.c [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+	int i=0;
+	data0=(int16_t**)malloc(Nline*sizeof(int16_t*));
+	for (i=0 ; i<Nline ; i++)
+	{
+		data0[i]=(int16_t*)malloc((buffer_length+1)*sizeof(int16_t));
+		data0[i][0]=(int16_t)(i+1);
+	}
+
+	load(buffer_length, Nline, data0, "hand.txt");
```
Comment (clecoued) [inline RedPitaya_emulator/srcbin/emul.c]: it would be nice to support a serie of images

File: RedPitaya_emulator/srcbin/emul.c [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+#include<stdlib.h>
+#include<stdint.h>
+#include<time.h>
+#include<signal.h>
+
+#include "TCP_API.h"
+
+#define PORT 7538
+
+float x0=80.0;
```
Comment (clecoued) [inline RedPitaya_emulator/srcbin/emul.c]: I think we should define and read a config file for probe settings associated with each images

**Comment by clecoued** [comment]
update run.sh to launch executable from linux/mac laptop or red pitaya

**Comment by GG23800** [comment]
Need to make the bash file now

File: RedPitaya_emulator/srcbin/emul.c (hunk 3 above)
Comment (kelu124) [reply RedPitaya_emulator/srcbin/emul.c]: And ideally list some settings as well - that could be used as well in the long run to identify the settings / variables / configs / ... that will need to be stored in a dicom.

**Review by clecoued — APPROVED** [review]

**Pull request merged**